common/download.go

```diff
@@ -115,13 +115,19 @@ func (d *DownloadClient) Get() (string, error) {
 	// Files when we don't copy the file are special cased.
 	var f *os.File
 	var finalPath string
+	sourcePath := ""
 	if url.Scheme == "file" && !d.config.CopyFile {
+		// This is a special case where we use a source file that already exists
+		// locally and we don't make a copy. Normally we would copy or download.
 		finalPath = url.Path
+		log.Printf("Using local file: %s", finalPath)
```

Comment:
Can you prefix this log message with `[DEBUG]` or `[INFO]`. We haven't hooked up our logutils to Packer yet but we use that as a magic string to do it.